examples/tutorials/rigid_object_primitive_tutorial.py

```diff
@@ -0,0 +1,198 @@
+# [setup]
+import math
+import os
+from pathlib import Path
+
+import cv2
+import magnum as mn
+import numpy as np
+
+import examples.tutorials.tutorial_functions as tutFuncs
+import habitat_sim
+import habitat_sim.utils.common as ut
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+data_path = os.path.join(dir_path, "../../data")
+output_path = os.path.join(dir_path, "rigid_object_primitive_tutorial_output/")
+
+
+def make_video_cv2(
+    observations, camera_res=[540, 720], prefix="", open_vid=True, multi_obs=False
+):
+    videodims = (camera_res[1], camera_res[0])
+    fourcc = cv2.VideoWriter_fourcc("m", "p", "4", "v")
```

Comment:
I am working on defining a common set of visualization / tutorials methods that will work in Notebooks and locally. I can draft a PR to merge those in.

Reply:
@Skylion007 Let's get those in and then rebase this PR to consume them. Happy to see this converted to Collab before merge if everything is in place for that transition.